Resources/Prototypes/Entities/Mobs/Species/reptilian.yml

```
@@ -109,16 +113,21 @@
       jumpsuit:
         sizeMaps:
           32:
-            sprite: Mobs/Species/Human/displacement.rsi
-            state: jumpsuit-female
     # Corvax-Digitigrade-Start
+            sprite: Corvax/Mobs/Species/displacement.rsi
```

я бы это перенёс обратно над комент выше и добавил вот такой комент
```suggestion
            sprite: Corvax/Mobs/Species/displacement.rsi # Corvax-Digitigrade       
```
такое отделение будет лучше